pymc3/backends/report.py

```diff
@@ -151,7 +175,8 @@ def _add_warnings(self, warnings, chain=None):
         warn_list.extend(warnings)
 
     def _log_summary(self):
-
+        if self._n_tune is not None and self._n_draws is not None and self._t_sampling is not None:
+            logger.info(f'Sampling {self.n_tune} tune and {self.n_draws} draw iterations took {self.t_sampling:.0f} seconds.')
```

Comment:
Can this include the number of chains and total number of draws? That might clarify how we count: For 1000 draws in 4 chains, the progressbar will go to 4000, so a message like "Sampling 500 tune and 1000 draws in 4 chains (2000 plus 4000 draws total) took 18 seconds." might be helpful.

What do you think?

Also, I might use `{self.n_tune:,d}` and `{self.n_draws:,d}` to get commas in the number, but that's a very weak desire.

Reply:
I'll add implement your suggestions tomorrow - thanks!

As a German, I read `10,000` as `float(10)`, but maybe we can go with the pythonic neutral ground of `10_000`?

Reply:
This is with the update I just pushed:

5 chains sequentially, interrupted during tuning of the first:
![image](https://user-images.githubusercontent.com/5894642/76334037-cc744100-62f2-11ea-84f7-6e66a5651897.png)

Interrupted during the second chain:
![image](https://user-images.githubusercontent.com/5894642/76334141-eb72d300-62f2-11ea-956a-e5e5959326dc.png)

Not interrupted:
![image](https://user-images.githubusercontent.com/5894642/76334222-05141a80-62f3-11ea-95ef-0acdc8f42040.png)

Reply:
haha! I thought there was a locale-safe version of this. this looks great to me -- feel free to merge.